src/main/resources/static/css/main.css

```
@@ -1,3 +1,9 @@
+h1 {
+font-family: sans-serif;
+font-size: 2em;
+text-transform: capitalize;
+}
```

It is better to format css too.

```suggestion
h1 {
    font-family: sans-serif;
    font-size: 2em;
    text-transform: capitalize;
}
```